Neos.Eel/Classes/Helper/ArrayHelper.php

```diff
@@ -351,11 +351,11 @@ public function push($array, $element): array
         if ($array instanceof \Traversable) {
             $array = iterator_to_array($array);
         }
-        if (is_scalar($array)) {
+        if (is_scalar($array) || $array === null) {
             $array = [$array];
         }
```

Comment:
I did this locally:

```suggestion
        if (is_scalar($array)) {
            $array = [$array];
        }
        if ($array === null) {
            $array = [];
        }
```

That way NULL does not end up in the array. This is more similar to the code replaced with #3654, where NULL would be skipped.

Reply:
I would also prefer @kdambekalns solution